qubes-rpc/qfile-unpacker.c

```diff
@@ -65,6 +65,7 @@ int main(int argc __attribute((__unused__)), char ** argv __attribute__((__unuse
 	switch (pid=fork()) {
 		case -1:
 			gui_fatal("Failed to create new process");
+			__attribute__ ((fallthrough));
```

Comment:
Better mark gui_fatal with `noreturn`. Or add (never reachable) `exit(1)` here. Otherwise this will be misleading.

Reply:
And also this breaks the build on older gcc (see travis linked below):
```
qfile-unpacker.c: In function 'main':
qfile-unpacker.c:68:4: error: empty declaration [-Werror]
    __attribute__ ((fallthrough));
    ^
cc1: all warnings being treated as errors
<builtin>: recipe for target 'qfile-unpacker.o' failed
```